net-news/newsboat/files/newsboat-2.13-json-c-0.14.0.patch

```diff
@@ -0,0 +1,34 @@
+From db70049f436a965767e8a1273389e1be92e946fa Mon Sep 17 00:00:00 2001
```

Comment:
Link to where this is from?

Reply:
This line references the commit in newsboat where the patch was merged (date is screwed up though): I'm not sure I understood what you wanted but modified the header with the github url of the merged pull request which applied the patch.

I removed the dependency commit, thanks for catching that up!